survey-sdk/src/main/java/co/nimblehq/survey/sdk/request/Credentials.kt

```diff
@@ -0,0 +1,6 @@
+package co.nimblehq.survey.sdk.request
+
+data class Credentials(val clientID: String, val clientSecret: String) {
+    constructor() : this("", "")
+}
```

Comment:
do you want to set default values? how about this?
```suggestion
data class Credentials(
    val clientID: String = "",
    val clientSecret: String = ""
)
```

Reply:
@luongvo I set de default constructor already. But seem like it can be more simple by set the default as your suggestion.